site/access-control.xml

```diff
@@ -282,10 +301,31 @@ limitations under the License.
 </pre>
 
           <p>
+            The example above uses an alias, <code>internal</code> for <code>rabbit_auth_backend_internal</code>.
+            The following aliases are available:
+
+            <ul>
+              <li><code>internal</code> for <code>rabbit_auth_backend_internal</code></li>
+              <li><code>ldap</code> for <code>rabbit_auth_backend_ldap</code> (from the <a href="/ldap.html">LDAP plugin</a>)</li>
+              <li><code>http</code> for <code>rabbit_auth_backend_http</code> (from the <a href="https://github.com/rabbitmq/rabbitmq-auth-backend-http">HTTP auth backend plugin</a>)</li>
+              <li><code>amqp</code> for <code>rabbit_auth_backend_amqp</code> (from the <a href="https://github.com/rabbitmq/rabbitmq-auth-backend-amqp">AMQP 0-9-1 auth backend plugin</a>)</li>
+              <li><code>dummy</code> for <code>rabbit_auth_backend_dummy</code></li>
```

Comment:
How are aliases managed? Is it a hard-coded list? Or does RabbitMQ try to use `rabbit_auth_backend_${alias}`?